src/genpy/message.py

```diff
@@ -106,10 +107,10 @@ def strify_message(val, indent='', time_offset=None, current_time=None, field_fi
     elif type_ in (int, long, float, bool):
         return str(val)
     elif isstring(val):
-        #TODO: need to escape strings correctly
         if not val:
             return "''"
-        return val
+        # escape strings for use in yaml file using yaml dump
+        return yaml.dump(val).split('\n')[0]
```

Comment:
Sorry about the late reply, must have over looked the notification while traveling...
You're right, I did not fix it for multi-line strings, yet.
My last commit now solves this as well.